tests/cstor/gtest/test_uzfs.cc

```diff
@@ -3342,13 +3365,24 @@ TEST(MgmtThreadTest, RebuildFailureWrongRebuildState) {
 	EXPECT_EQ(status, ZVOL_OP_STATUS_FAILED);
 }
 
-/* Rebuild success, Single replica success */
+/* Rebuild failed, Single replica */
 TEST(MgmtThreadTest, RebuildFailureSingleReplica) {
 	uzfs_mgmt_conn_t *conn = (uzfs_mgmt_conn_t *)zinfo->mgmt_conn;
 	mgmt_thread_test_case(24);
 	EXPECT_EQ(status, ZVOL_OP_STATUS_FAILED); // errout on healthy replica
 }
 
+/*
+ * Rebuild success, Single replica case when volumes are healthy but
+ * quorum is off. Another test case i.e replica is healthy and quorum on
+ * is covered in test_zrepl_prot
+ */
+TEST(MgmtThreadTest, RebuildSuccessSingleReplica) {
+	uzfs_mgmt_conn_t *conn = (uzfs_mgmt_conn_t *)zinfo->mgmt_conn;
+	mgmt_thread_test_case(25);
+	EXPECT_EQ(status, ZVOL_OP_STATUS_FAILED);
```

Comment:
there are a lot of test cases which rebuilds pool1/vol1 and once rebuilding is done, quorum will be set. Probably that is the reason why travis is failing. At this stage, zinfo has quorum on and returning successful for REBUILD_START opcode.  Probably set the uzfs_zinfo_set_quorum to off before invoking this to guaranty that it is a non-quorum replica.

Reply:
Set quorum to off before starting test